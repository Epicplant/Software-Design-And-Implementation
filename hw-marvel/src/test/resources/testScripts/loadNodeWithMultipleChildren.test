# Tests that a graph with 2 or more connected nodes loads properly

# Loads a new graph from a csv file
LoadGraph newGraph complexHeroGraph.csv

# Lists all of the nodes of this new graph
ListNodes newGraph

# Lists all of the children of A
ListChildren newGraph A

# Lists all of the children of B
ListChildren newGraph B