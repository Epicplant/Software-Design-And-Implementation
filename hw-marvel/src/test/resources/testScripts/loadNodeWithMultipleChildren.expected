# Tests that a graph with 2 or more connected nodes loads properly

# Loads a new graph from a csv file
loaded graph newGraph

# Lists all of the nodes of this new graph
newGraph contains: A B C D E F G H I

# Lists all of the children of A
the children of A in newGraph are: B(1) H(1)

# Lists all of the children of B
the children of B in newGraph are: A(1) C(2) D(2) H(1)